src/taco-livereload/lib/utils/homePage/bower_components/q/.npmignore

```diff
@@ -0,0 +1,23 @@
+node_modules
```

Comment:
@MSLaguana On second thoughts, I think we should package this code with the html code (it's standard practice). Otherwise, the end-user will need to be connected to the internet in order to use livereload (which is not a desirable constraint, IMO).